### ISSUE TITLE
How to perform edit using demo? 

### ISSUE THREAD

**Issue by connorzl** [amirhertz/spaghetti#2]
Hi, thanks for the cool demo! I'm trying to play around and edit parts -- I can currently select parts but pressing 'G' does not seem to allow me to translate my selected part. Do I have to press some other buttons or click/drag on the mouse a certain way?

**Comment by connorzl** [comment]
Also, is the shape supposed to appear in the larger blue window?

**Comment by amirhertz** [comment]
Yes, after each selection, a new shape should appear in the main blue window. If not, I would check the print in the console, are there any errors? what is the last message? 
Afterward, the edit can be made in the main window.

**Comment by connorzl** [comment]
![image](https://user-images.githubusercontent.com/14813758/168543049-0e875d2d-4c0c-497f-9df2-017b3d2d58e8.png)

The above is what I see, and I get errors in the console: 

![image](https://user-images.githubusercontent.com/14813758/168543153-04406ee6-42f2-4765-85d9-0c5d1817edcd.png)

**Comment by connorzl** [comment]
Update -- I was able to get the demo working by running it on an Ubuntu machine instead of my mac laptop

**Comment by amirhertz** [comment]
Great! thanks for the update.
Unfortunley, I have _only_ Ubuntu and Windows on my disposal and on both the demo should work.
Do you know why it doesn't work on mac? I will be happy do make or pull the neccseray fixes.
